shared/attributes.asciidoc

```diff
@@ -60,6 +59,10 @@
 :curator-ref:          https://www.elastic.co/guide/en/elasticsearch/client/curator/{branch}
 :curator-ref-current:  https://www.elastic.co/guide/en/elasticsearch/client/curator/current
 :infra-guide:          https://www.elastic.co/guide/en/infrastructure/guide/{branch}
```

Comment:
I wonder if we should create a new "LEGACY" section in this file and move this attribute to it. That would help prevent someone from accidentally trying to use this attribute in the future.

Reply:
That's not a bad idea, although if someone tries to use this attribute, the build will break anyway, as the file doesn't exist any more. But I'm all for clarity up front.